src/main/java/com/keniu/models/CartItem.java

```diff
@@ -0,0 +1,41 @@
+package com.keniu.models;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+
+@Entity
+@SQLDelete(sql = "UPDATE cart_items SET is_deleted = true WHERE id=?")
+@SQLRestriction("is_deleted = false")
+@Getter
+@Setter
+@Table(name = "cart_items")
+public class CartItem {
```

Comment:
Make fetchType = LAZY for all @ManyToOne relationships and use @Query with JOIN FETCH in your repository methods if you need to load an addtional data

Reply:
Ok, done